apps/agreement/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@aragon/apps-agreement",
+  "version": "0.1.0",
+  "author": "Aragon Association <legal@aragon.org>",
+  "license": "(GPL-3.0-or-later OR AGPL-3.0-or-later)",
+  "files": [
+    "/abi",
+    "/build",
+    "/contracts",
+    "/test"
+  ],
+  "scripts": {
+    "compile": "truffle compile",
+    "lint": "solium --dir ./contracts",
+    "test": "TRUFFLE_TEST=true npm run ganache-cli:test",
+    "test:gas": "GAS_REPORTER=true npm test",
+    "coverage": "SOLIDITY_COVERAGE=true npm run ganache-cli:test",
+    "ganache-cli:test": "./scripts/ganache-cli.sh",
+    "abi:extract": "truffle-extract --output abi/ --keys abi",
+    "prepublishOnly": "truffle compile --all && npm run abi:extract -- --no-compile",
+    "apm:prepublish": "npm run compile",
+    "apm:publish:major": "aragon apm publish major --files public/ --prepublish-script apm:prepublish",
+    "apm:publish:minor": "aragon apm publish minor --files public/ --prepublish-script apm:prepublish",
+    "apm:publish:patch": "aragon apm publish patch --files public/ --prepublish-script apm:prepublish"
+  },
+  "dependencies": {
+    "@aragon/os": "aragon/aragonOS#add_disputable_base_app"
+  },
+  "devDependencies": {
+    "@aragon/apps-shared-migrations": "1.0.0",
+    "@aragon/cli": "^7.1.3",
+    "@aragon/minime": "^1.0.0",
+    "@aragon/contract-test-helpers": "^0.0.1",
+    "@aragon/truffle-config-v5": "^1.0.0",
+    "eth-gas-reporter": "^0.2.0",
+    "ethereumjs-testrpc-sc": "^6.5.1-sc.0",
```

Comment:
Quick note, I think we can remove this now, since `solidity-coverage@^0.7.0` doesn't need it anymore!